tests/acceptance/acceptancesuite.js

```diff
@@ -105,3 +106,65 @@ test(`Facets load on the page, and can affect the search`, async t => {
   actualResultsCount = await verticalResultsComponent.getResultsCountTotal();
   await t.expect(actualResultsCount).eql(initialResultsCount);
 });
+
+fixture`Experience links work as expected`
+  .before(setupServer)
+  .after(shutdownServer)
+  .page`http://localhost:9999/tests/acceptance/fixtures/html/facets`;
+
+test('Facets, pagination, and filters do not persist accross experience links', async t => {
```

Comment:
Are we verifying that static filters and sorts are also not preserved in this test?

Reply:
Added